documentation/book/security.adoc

```diff
@@ -0,0 +1,64 @@
+== Security
+
+The Apache Kafka project supports data encryption by means of the SSL/TLS protocol.
+This makes it possible to encrypt data transferred between brokers (interbroker communication) and between clients and brokers.
+Leveraging the SSL/TLS support, it is also possible to have mutual authentication, where the Kafka broker authenticates the client by verifying its certificate, and the client does likewise with the server's certificate.
```

Comment:
Do we actually support this in this PR?

Reply:
No, it's out of the scope for this PR. Do you think we should remove this statement because it sounds that we are going to support client auth right now?

Reply:
That is how I would interpret it ... that we support TLC client auth.